files/lib/system/discord/DiscordApi.class.php

```diff
@@ -1218,9 +1218,13 @@ public function createGuild($params)
      *
      * @return  array
      */
-    public function getGuild()
+    public function getGuild(bool $counts = false)
```

Comment:
Please rename `$counts` to `$withCounts`, because the parameter in documentation is know as `with_counts `